src/components/RegistrationSuccessForm/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./styles.scss";
+
+export default function RegistrationSuccessForm() {
```

Comment:
IMO, I'm comfortable with the word **Sign up** and **Sign in**. Should we stick we these words? @kimvcp @Raksani

Reply:
I'd like it to be 'SignUpSuccess[screen? modal?]' without the word 'form' because there's no form for users to fill, only just to show the progress (failed/success) ☺️